charts/stable/qbittorrent/templates/common.yaml

```diff
@@ -3,7 +3,7 @@
 
 {{/* Disable [qbitportforward] if requested */}}
 {{- if not .Values.qbitportforward.enabled -}}
-  {{- $_ := set .Values.workload.qbitportforward "enabled" false -}}
+  {{- $_ := set qbittorrent.cronjob "enabled" false -}}
```

Comment:
The value after set is a yamlpath to values, or a variable.
So this won't work, what you wanted here is an include probably.
But as I mentioned in the other comment, you don't need to change this really